wagtail/admin/templates/wagtailadmin/shared/editing_sessions/list.html

```diff
@@ -0,0 +1,124 @@
+                    {{ avatar }}
+                    <div class="w-editing-sessions__popup" data-w-tooltip-target="content" hidden>
+                        <span class="w-editing-sessions__name">{{ session.user|user_display_name|default:_("system")|capfirst }}</span>
+                        <span class="w-sr-only">-</span>
+                        <span>{% trans "Currently viewing" %}</span>
+                    </div>
+                </button>
+            {% endif %}
+        </li>
+    {% endfor %}
+    {% if sessions|length > 4 %}
+        {% blocktranslate trimmed count num_user=sessions|length|add:"-4" asvar num_other_users_message %}
+            One other user is currently viewing
+        {% plural %}
+            {{ num_user }} other users are currently viewing
+        {% endblocktranslate %}
+
+        {% fragment as more_sessions_toggle %}
+            +{{ sessions|length|add:"-4" }}
+        {% endfragment %}
+
+        {% dropdown theme="drilldown" classname="w-editing-sessions__session w-editing-sessions__session--more" toggle_classname="w-editing-sessions__avatar" toggle_label=more_sessions_toggle toggle_aria_label=num_other_users_message|capfirst %}
+            <ol class="w-editing-sessions__more-list">
+                {% for session in sessions|slice:"4:" %}
+                    <li class="w-editing-sessions__list-item">
+                        {% avatar user=session.user size="small" %}
+                        <span>{{ session.user|user_display_name|default:_("system")|capfirst }}</span>
+                    </li>
+                {% endfor %}
+            </ol>
+        {% enddropdown %}
+    {% endif %}
+</ol>
+
+{% if sessions.0.revision_id %}
+    <template data-controller="w-teleport" data-w-teleport-target-value="#title-text-w-overwrite-changes-dialog" data-w-teleport-reset-value="true">
+        {% if request.user == sessions.0.user %}
+            {% blocktranslate trimmed with model_name=content_type.name asvar someone_has_saved_message %}
+                You have saved a newer version of this {{ model_name }} in another session
+            {% endblocktranslate %}
+        {% else %}
+            {% blocktranslate trimmed with user_name=sessions.0.user|user_display_name|default:_("system") model_name=content_type.name asvar someone_has_saved_message %}
+                {{ user_name }} has saved a newer version of this {{ model_name }}
+            {% endblocktranslate %}
+        {% endif %}
+        {{ someone_has_saved_message|capfirst }}
+    </template>
+
+    <template data-controller="w-teleport" data-w-teleport-target-value="#subtitle-w-overwrite-changes-dialog" data-w-teleport-reset-value="true">
+        {% if request.user == sessions.0.user %}
+            {% blocktranslate trimmed asvar overwrite_message %}
+                Proceeding will overwrite the changes you made in that session. Refreshing the page will lose any of your unsaved changes in the current session.
```

Comment:
Re the wording – "that session"? Maybe replace with "another session"?

Reply:
This is the subtitle. The title already says "You have saved a newer version of this {{ model_name }} in another session", I think it doesn't read very well if we repeat "another session":

<img width="590" alt="You have saved a newer version of this page in another session
Proceeding will overwrite the changes you made in that session. Refreshing the page will lose any of your unsaved changes in the current session." src="https://github.com/user-attachments/assets/3ffc1ec8-54ad-41d4-8411-e6dc8a977d76">

Reply:
Thank you, I had missed that